src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -380,11 +380,20 @@ type MessageDbCategory<'event, 'state, 'context>(resolveInner, empty) =
 
 
 type MessageDbConnector(
-    connectionString: string, readOnlyConnectionString: string,
+    connectionString: string, ?readOnlyConnectionString: string,
     [<O; D(null)>]?readRetryPolicy, [<O; D(null)>]?writeRetryPolicy) =
         member _.Connect() =
-            let writeConnection = NpgsqlDataSource.Create(connectionString)
-            let readConnection = NpgsqlDataSource.Create(readOnlyConnectionString)
+            let connectToDb connectionString () =
+                task {
```

Comment:
I know this will be uncomfortable, but an unwritten rule in equinox is my consistent use of the debatable convention of puttin g the `= task {` on the previous line and unindenting
Also if there's a version of `OpenAsync` that takes a CT, I thread those through wherever possible

Reply:
Totally get it, with 4 spaces of indentation you need to justify each extra indentation level 👍

Reply:
:D (2 spaces just makes it too haskelly for me!)